app/stylesheets/boilerplate/logo.scss

```diff
@@ -0,0 +1,24 @@
+@import '../config/type';
+@import '../config/colours';
+@import '../config/dimensions';
+
+.header {
+
+  img {
+      float: left;
+      margin-right: $header-common-margin;
+      margin-top: $logo-top-position;
+      width: $header-height;
+    }
+    h1 {
+      color: $theme-blue-alt;
+      display: inline-block;
+      float: left;
+      font-size: $dash-title;
+      font-weight: $title-weight;
+      line-height: $dash-title + 2;
```

Comment:
ok for the title to have this `+ 2`, but for other text we can keep `line-height` variables in `type.scss`